core/src/client/async_client/mod.rs

```diff
@@ -394,6 +381,7 @@ impl SubscriptionClientT for Client {
 	///
 	/// The `subscribe_method` and `params` are used to ask for the subscription towards the
 	/// server. The `unsubscribe_method` is used to close the subscription.
+	#[instrument(name = "subscription", skip(self, params), level = "trace")]
```

Comment:
note this differs a bit from the servers where don't distinguish between subscriptions and calls because it's so messy to support.

do we want both `subscribe and unsubscribe method`? here or only subscribe?